service/collector/elb.go

```diff
@@ -120,20 +120,27 @@ func (e *ELB) collectForAccount(ch chan<- prometheus.Metric, awsClients clientaw
 		return microerror.Mask(err)
 	}
 
-	var loadbalancers []*elb.LoadBalancerDescription
+	var loadBalancers []*elb.LoadBalancerDescription
 	{
 		i := &elb.DescribeLoadBalancersInput{}
 		o, err := awsClients.ELB.DescribeLoadBalancers(i)
 		if err != nil {
 			return microerror.Mask(err)
 		}
-		loadbalancers = o.LoadBalancerDescriptions
+		loadBalancers = o.LoadBalancerDescriptions
+
+		if len(loadBalancers) == 0 {
```

Comment:
This is the actual fix in here. Skip further actions when there are no load balancers.

Reply:
Do I understand this correctly that in this case there are no clusters in the whole account? Otherwise I would have expected a `continue` and not a `return`.